src/handleSidebarProjects.js

```diff
@@ -0,0 +1,20 @@
+import { projects } from "./projectsData";
+
+const sideBarProjects = document.getElementById("sidebar-projects-container");
+
+export function renderSidebarProjects() {
+  const fragment = new DocumentFragment();
+  projects.forEach((project) => {
```

Comment:
Use map and object destructuring to make it more consice.

like `const projectElements = projects.map(({ name }) => {`